### PR TITLE
UI enhancements to the incident search view

### PR DESCRIPTION
**Pull request by jdc91**
UI enhancements to the incident search view

### CODE REVIEW

**Comment by jdc91** [comment]
@MrClemRkz  Please review if working after merge

File: frontend/src/modules/ongoing-incidents/state/OngoingIncidents.actions.js [hunk 1]
```diff
@@ -37,347 +37,364 @@ import {
...
-        data: null,
-        error:error,
+  return {
+    type: REQUEST_INCIDENT_ESCALATE_ERROR,
+    data: null,
+    error: error
+  };
+}
+
+export function fetchEscallateIncident(incidentId, assigneeId) {
```
[the remark this answers is not in the document]
Comment (jdc91) [reply frontend/src/modules/ongoing-incidents/state/OngoingIncidents.actions.js]: @lahiru94 @ManZzup Can you check which is the correct one here?

[the remark this answers is not in the document]
Comment (jdc91) [reply frontend/src/modules/ongoing-incidents/state/OngoingIncidents.actions.js]: Fixed issue, please merge code :)

[the remark this answers is not in the document]
Comment (MrClemRkz) [reply frontend/src/modules/ongoing-incidents/state/OngoingIncidents.actions.js]: thank you very much @jdc91. 

tested and confirmed.

**Pull request merged**